packages/common/src/enum/milestone-status.ts

```diff
@@ -121,3 +121,48 @@ export enum StatusId {
   SENT_EMPLOYER_DOCUMENTS_TO_HMBC = '1651cad1-1e56-4c79-92ce-f548ad9ec52c',
   WITHDREW_FROM_COMPETITION = '3fd4f2b0-5151-d7c8-6bbc-3a0601b5e1b0',
 }
+
+export enum STATUS_REASONS {
+  COST_OF_PROCESS = 'Cost of the IEN process',
+  TIMING = 'Timing',
+  WITHDREW_FROM_PROGRAM = 'Withdrew from IEN program',
+  PURSUING_LICENSE_IN_ANOTHER_PROVINCE = 'Pursuing licensure in another province',
+  ACCPTED_JOB_OUTSIDE_BC = 'Accepted another job outside of BC',
+  ACCEPTED_JOB_IN_BC = 'Accepted another job in BC',
+  ACCPTED_NURSING_JOB_OUTSIDE_BC = 'Accepted another nursing job (public or private) outside of BC',
+  ACCEPTED_NURSING_JOB_IN_BC_PUBLIC = 'Accepted another nursing job (public) in BC',
+  ACCEPTED_NURSING_JOB_IN_BC_PRIVATE = 'Accepted another nursing job (private) in BC',
+  JOB_SEARCH_ON_HOLD = 'Job search on hold',
+  COST_OF_LIVING = 'Community - Cost of living',
+  OTHER = 'Other',
+  COMPENSATION = 'Compensation (pay/benefits)',
+  REVOKED_JOB_SEARCH = 'Revoked job search',
+  UNRESPONSIVE = 'Unresponsive',
+  LIMITED_ACCOMODATION = 'Community - Limited accommodation/housing available',
+  JOB_SUITABILITY = 'Job suitability',
+  PERSONAL_REASONS = 'Personal reasons',
+  CANNOT_MEET_PROGRAM_CRITERIA = 'Cannot meet the program criteria',
+  IMMIGRATION_ISSUE = 'Immigration concerns/issues',
+}
+export enum CandidateWithdrawReasonId {
+  COST_OF_PROCESS = 'FF4B82C9-1E4D-4561-84AA-08EB2DD9D37E',
+  TIMING = 'BFFAFBFD-EDFD-4F62-B72B-25E33BB55992',
+  WITHDREW_FROM_PROGRAM = 'A1E0E2C1-E272-468A-8F7A-28686B750CAE',
+  PURSUING_LICENSE_IN_ANOTHER_PROVINCE = 'E0930E51-630A-474F-BAE1-31E52A552CBB',
+  ACCPTED_JOB_OUTSIDE_BC = '74905D23-EF3D-453C-A2F2-45E4E2798A0B',
+  ACCEPTED_JOB_IN_BC = '1886B20E-17FF-41E6-8915-ECF6E03900DB',
+  ACCPTED_NURSING_JOB_OUTSIDE_BC = '3F84D93D-C122-4895-8D49-4D975F0D0005',
+  ACCEPTED_NURSING_JOB_IN_BC_PUBLIC = 'B1E993D1-58EC-4FE5-A31B-532E070C805B',
+  ACCEPTED_NURSING_JOB_IN_BC_PRIVATE = 'F281B9A1-E1F7-4984-9CD1-ECF805819926',
+  JOB_SEARCH_ON_HOLD = '579AF591-F009-4E8F-806A-61EB3920CA1B',
+  COST_OF_LIVING = 'AFC1B84C-682D-4F4A-A624-65174A74F02D',
+  OTHER = 'E2958403-CB56-4396-B903-654C1FE787CC',
+  COMPENSATION = '583C72E3-6D58-4E0B-ADC1-7F5E89A62CD5',
+  REVOKED_JOB_SEARCH = 'C09A8B50-7FE5-4EED-84E5-810CCB17F5D4',
+  UNRESPONSIVE = 'ED497BF5-4E1D-46A0-852B-921628125839',
+  LIMITED_ACCOMODATION = 'C9C6BD85-35AC-443D-91D5-C488594622AB',
+  JOB_SUITABILITY = 'FD756221-0F02-4484-9F8F-DB78E94E78EB',
+  PERSONAL_REASONS = '49873A72-AB2D-4349-AD4D-EE49CEB8288F',
+  CANNOT_MEET_PROGRAM_CRITERIA = '1824580D-ED74-4E37-BA1C-F15DBB3878D2',
+  IMMIGRATION_ISSUE = '031CBA24-8B47-4AD8-A932-FA2FA858B25B',
+}
```

Comment:
are the queries case-sensitive? my db has them all in lower case

Reply:
They don't seem to be, but I can make these lower case for consistency